buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -141,14 +142,20 @@ abstract class AndroidTarget extends JavaLibTarget {
 
     @Override
     Scope getApt() {
-        return Scope.from(project, baseVariant.annotationProcessorConfiguration)
+        if (!isKotlin)
+            return Scope.from(project, baseVariant.annotationProcessorConfiguration)
+        else
+            return Scope.from(project, KotlinLibTarget.ANNOTATION_PROCESSOR_CONFIGURATION_NAME)
     }
 
     @Override
     Scope getTestApt() {
-        return Scope.from(project, unitTestVariant
-                ? unitTestVariant.annotationProcessorConfiguration : null
-        )
+        if (!isKotlin)
+            return Scope.from(project, unitTestVariant
+                    ? unitTestVariant.annotationProcessorConfiguration : null
+            )
+        else
+            return Scope.from(project, KotlinLibTarget.ANNOTATION_PROCESSOR_CONFIGURATION_NAME)
```

Comment:
Could you add a test to make sure that UnitTests in modules with KAPT work? (if one doesn't already exist)